scripts/dataflow_delete.py

```diff
@@ -30,7 +31,7 @@
     copied_older_than = copied_older_than_days * num_secs_in_a_day # seconds
 
 else:
-    threshold = 0.95
+    threshold = 0.85
```

Comment:
maybe declare this as a constant at the top of the program so the special value is easy to see?

Reply:
Or specify in the config so that it's user configurable.

Reply:
Moved to config, thanks!